src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -208,6 +211,18 @@ export default class OncoprintControls extends React.Component<IOncoprintControl
         this.onHorzZoomSliderSet = this.onHorzZoomSliderSet.bind(this);
         this.onSetHorzZoomTextInput = this.onSetHorzZoomTextInput.bind(this);
 
+        // add pre-selected treatments (passed in via URL in ResultsViewOncoprint) to selected treatments
+        this.initSelectedTreatments = autorun(() => {
```

Comment:
i think we should be able to do this without autorun.    Isn't _selectedTreatmentOptionsObsArray just a computation of  treatmentOptionsByValueMap and this.props.state?

Reply:
I tried this at first, but any solution that used @computed does not allow the _de_selection of treatments. The autorun solution correctly handles init of the selected treatments.

Reply:
Autorun has been removed. Treatment selections and options are passed in as props.